packages/esm-patient-vitals-app/src/vitals-and-biometrics-header/vitals-header.component.tsx

```diff
@@ -57,6 +59,23 @@ const VitalsHeader: React.FC<VitalsHeaderProps> = ({ patientUuid }) => {
     const hasActiveVisit = Boolean(currentVisit?.uuid);
     const vitalsTakenToday = Boolean(dayjs(latestVitals?.date).isToday());
```

Comment:
We don't need to handle the case where vitals are collected today because it will be taken care of in this condition. The tag won't get rendered at all.